pkg/controller/wildflyserver/wildflyserver_controller.go

```diff
@@ -319,7 +546,9 @@ func (r *ReconcileWildFlyServer) statefulSetForWildFly(w *wildflyv1alpha1.WildFl
 			Labels:    ls,
 		},
 		Spec: appsv1.StatefulSetSpec{
-			Replicas: &replicas,
+			Replicas:            &replicas,
+			ServiceName:         loadBalancerServiceName(w),
+			PodManagementPolicy: appsv1.ParallelPodManagement, // TODO: is it fine to change for Parralel?
```

Comment:
I'm not sure.
One of the feature of the stateful set is to start one pod after the other. This simplifies clustering code and I'm not sure it should be changed. This can be explored and tested in a future PR